.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
         - os: linux
           python: 3.5
         - os: linux
+          python: 3.6
+        - os: linux
```

Comment:
I believe so: https://travis-ci.org/zopefoundation/zope.index/builds/176093903 failed on OSX because "homebrew 3" is 3.6 already.